deployment/kubernetes/yaml/mongodb.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-mongodb
+spec:
+  selector:
+    app: ftgo-mongodb
+  ports:
+  - port: 27017
+    targetPort: 27017
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: ftgo-mongodb
+spec:
+  selector:
+    matchLabels:
+      app: ftgo-mongodb
+  replicas: 1
+  serviceName: ftgo-mongodb
+  template:
+    metadata:
+      labels:
+        app: ftgo-mongodb
+    spec:
+      terminationGracePeriodSeconds: 10
+      containers:
+      - name: ftgo-mongodb
+        image: mongo:4.4.0-rc8-bionic

```

Comment:
Please use the mongo image in the `docker-compos....` file.

Reply:
updated